### PR TITLE
cdc: add changefeed level config about delete-only-output-handle-key-columns

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/14182
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -130,11 +130,17 @@ write-key-threshold = 0
 # ]
 
 # The protocol configuration item specifies the protocol format of the messages sent to the downstream.
-# When the downstream is Kafka, the protocol can only be canal-json or avro.
+# When the downstream is Kafka, the protocol can only be canal-json, avro or open-protocol.
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:133]:
```suggestion
# When the downstream is Kafka, the protocol can only be canal-json, avro, or open-protocol.
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -130,11 +130,17 @@ write-key-threshold = 0
...
 # When the downstream is a storage service, the protocol can only be canal-json or csv.
 # Note: This configuration item only takes effect if the downstream is Kafka or a storage service.
 # protocol = "canal-json"
 
+# Starting from v7.2.0, the delete-only-output-handle-key-columns parameter is used to specify the output content of the Delete event, and it is only valid for the canal-json and open-protocol protocols.
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:138]:
```suggestion
# Starting from v7.2.0, the delete-only-output-handle-key-columns parameter specifies the output of Delete events. This parameter is valid only for canal-json and open-protocol protocols.
```

File: ticdc/ticdc-changefeed-config.md [hunk 3]
```diff
@@ -130,11 +130,17 @@ write-key-threshold = 0
...
+# The default value is false, which means outputting the contents of all columns. When set to true, only the contents of primary key columns or unique index columns # are output.
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:139]:
```suggestion
# The default value is false, which means outputting all columns. When you set it to true, only the primary key columns or unique index columns are output.
```

File: ticdc/ticdc-changefeed-config.md [hunk 4]
```diff
@@ -130,11 +130,17 @@ write-key-threshold = 0
 # ]
 
 # The protocol configuration item specifies the protocol format of the messages sent to the downstream.
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:132]:
```suggestion
# The protocol configuration item specifies the protocol format used when encoding messages.
```

File: ticdc/ticdc-changefeed-config.md [hunk 5]
```diff
@@ -130,11 +130,17 @@ write-key-threshold = 0
...
-# When the downstream is Kafka, the protocol can only be canal-json or avro.
+# When the downstream is Kafka, the protocol can only be canal-json, avro or open-protocol.
 # When the downstream is a storage service, the protocol can only be canal-json or csv.
 # Note: This configuration item only takes effect if the downstream is Kafka or a storage service.
 # protocol = "canal-json"
 
+# Starting from v7.2.0, the delete-only-output-handle-key-columns parameter is used to specify the output content of the Delete event, and it is only valid for the canal-json and open-protocol protocols.
+# The default value is false, which means outputting the contents of all columns. When set to true, only the contents of primary key columns or unique index columns # are output.
+# The Avro protocol is not controlled by this parameter and always outputs only the contents of primary key columns or unique index columns.
+# The CSV protocol is not controlled by this parameter and always outputs the contents of all columns.
```
Comment (Oreoxmt) [inline ticdc/ticdc-changefeed-config.md:141]:
```suggestion
# The Avro protocol is not controlled by this parameter and always outputs only the primary key columns or unique index columns.
# The CSV protocol is not controlled by this parameter and always outputs all columns.
```

File: ticdc/ticdc-changefeed-config.md [hunk 6]
```diff
@@ -129,12 +129,18 @@ write-key-threshold = 0
 #     {matcher = ['test6.*'], partition = "ts"}
 # ]
 
-# The protocol configuration item specifies the protocol format of the messages sent to the downstream.
-# When the downstream is Kafka, the protocol can only be canal-json or avro.
+# The protocol configuration item specifies the protocol format used when encoding messages.
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:132]:
```suggestion
# The protocol configuration item specifies the protocol format used for encoding messages.
```

File: ticdc/ticdc-changefeed-config.md [hunk 7]
```diff
@@ -129,12 +129,18 @@ write-key-threshold = 0
...
+# When the downstream is Kafka, the protocol can only be canal-json, avro, or open-protocol.
 # When the downstream is a storage service, the protocol can only be canal-json or csv.
 # Note: This configuration item only takes effect if the downstream is Kafka or a storage service.
 # protocol = "canal-json"
 
+# Starting from v7.2.0, the delete-only-output-handle-key-columns parameter specifies the output of Delete events. This parameter is valid only for canal-json and open-protocol protocols.
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:138]:
```suggestion
# Starting from v7.2.0, the `delete-only-output-handle-key-columns` parameter specifies the output of DELETE events. This parameter is valid only for canal-json and open-protocol protocols.
```

File: ticdc/ticdc-changefeed-config.md [hunk 8]
```diff
@@ -129,12 +129,18 @@ write-key-threshold = 0
...
+# The default value is false, which means outputting all columns. When you set it to true, only the primary key columns or unique index columns are output.
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:139]:
```suggestion
# The default value is false, which means outputting all columns. When you set it to true, only primary key columns or unique index columns are output.
```

**Review by qiancai — APPROVED** [review]
/lgtm

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**